src/corona_hakab_model/consts.py

```diff
@@ -102,9 +71,42 @@
         lambda agent: agent.medical_state.name == "Recovered",
         lambda agent: agent.medical_state.name == "Symptomatic",
     ),  # TODO: Define better API
-    # dictionary of {date : percent} that controls what percentage of schools are open
-    "school_openage_factors": {1: 0, 30: 0.5, 45: 1},
-    "should_change_school_openage": False
+
+    # policies data
+    # a dictionary of type day:List[ConnectionTypes]. on each day, keeps only the given connection types opened
+    "policies_changes": {
```

Comment:
no dicts in const! use a tuple of tuples instead

Reply:
why? it would make things a lot uglier.
if this is un-avoidable, is there a not-ugly way too check item existence in a tuple of tuples?
(i would like to know if a given date is in this data structure, not by looping through each item)

Reply:
What I would do is export these parameters to another place, since they are not Constant values ever present in the simulation, but parameters for the simulation.

The medical state parameters are going to be detached from the consts soon enough too

Reply:
good idea, i agree.
i think there should be a consts file for each different aspect of the simulation (medical consts, policy consts, plotting, tests, population and etc...)
but just for now, what is the problem with putting a dict in consts? i would rather not switch it to tuples before the version release.